Epam.TestAutomation.Web/PageObjects/Pages/ListOfLanguagesPage.cs

```diff
@@ -0,0 +1,10 @@
+using Epam.TestAutomation.Core.BasePage;
+
+namespace Epam.TestAutomation.Web.PageObjects.Pages;
+
+public class ListOfLanguagesPage : BasePage
```

Comment:
It is not a page. It is like a block on the main page so change it to the block and create the instance of this class in the MainPage class.